src/lib/utils/options/sources/typedoc.ts

```diff
@@ -404,6 +404,11 @@ export function addTypeDocOptions(options: Pick<Options, "addDeclaration">) {
         help: "Do not print the TypeDoc link at the end of the page.",
         type: ParameterType.Boolean,
     });
+    options.addDeclaration({
+        name: "footer",
+        help: "Set custom footer when hideGenerator option is set to true",
```

Comment:
I think it would be better to name this `customFooterHtml` to be consistent with the `customCss` option, also to hint that this option is just raw HTML that will be injected into the page